src/components/TextInput/__tests__/TextInput.spec.js

```diff
@@ -150,6 +150,39 @@ describe('Text input', () => {
     expect(textInput.value).toBe('');
   });
 
+  describe('with tooltip', () => {
+
+    const propsTooltip = {
+      ...initialProps,
+      label: 'company',
+      tooltipContent: 'enter your company'
+    };
+
+    const propsTooltipLeading = {
+      ...propsTooltip,
+      tooltipPosition: 'leading'
+    };
+
+    it('the input can be selected by getByLabelText when the tooltip is trailing', async () => {
```

Comment:
nit: the naming of this and the following test is a little odd in that it sounds like it's asserting that we can use the test framework rather than explaining what it is we're actually testing in the component.  I'm guessing what we're trying to test it really more like "the label is correctly associated with the input" rather than "the input can be selected by <test framework function>".

Reply:
yes indeed!